docs/source/developing/developing.adoc

```diff
@@ -20,7 +20,7 @@ Mac OS X or Windows. The Windows operating system might require additional steps
 have some limitations.
 
 [[set-up-dev-env]]
-== Setting up the development environment
+== Workspace creation
```

Comment:
Actually "Setting up the development environment" sounds better to me i.e. I mean may be in India we use it more. @Preeticp comments?